### ISSUE TITLE
JSON-LD template for dams

### ISSUE THREAD

**Issue by ksonda** [internetofwater/docs.geoconnex.us#4]
Of interest to @mikejohnson51 and [ORNL/Hydrosource](https://github.com/internetofwater/geoconnex-hydrosource)

**Comment by ksonda** [comment]
should be developed as part of [internetofwater/geoconnex.us#11](https://github.com/internetofwater/geoconnex.us/issues/139)

**Comment by ksonda** [comment]
In lite of recent developments. @dblodgett-usgs, you and I should do this. I imagine its basically the same as the surface feature thing you did already, but add some reference to impounded water body . We also still need to to draft something for parameter/period of record

**Comment by dblodgett-usgs** [comment]
90% of it is the hydrologic location that we already have in the wiki. The hydroLocationType would be "dam" per: https://docs.opengeospatial.org/is/14-111r6/14-111r6.html#_the_river_referencing_model

Do we have waterbodies / reservoirs to point to? If we do, then starting to link some of that stuff with the storage model would be appropriate, but I don't know how much content we can muster? https://docs.opengeospatial.org/is/14-111r6/14-111r6.html#_the_storage_model

Note that in the storage model, the waterbody is where you would put stuff like bathymetry or a polygon for a shoreline. The reservoir is where you would be stuff like operating rules or storage curves.

**Comment by ksonda** [comment]
fair enough. ORNL has linked ~1300 hydropower dams to nhd waterbodies. https://hydrosource.ornl.gov/dataset/hydropower-infrastructure-lakes-reservoirs-and-rivers-HILARRI
